source/manuals/programming-languages/nodejs/style-guide.html.md.erb

```diff
@@ -0,0 +1,122 @@
+---
+title: GDS Node.js Style Guide
+---
+
+# <%= current_page.data.title %>
+
+## General style
+
+As with all JavaScript at GDS, for Node.js applications we use the [standard] style.
+
+## Use strict mode
+
+[Strict mode] should be enabled for Node.js modules, as in the browser this prevents polluting the global scope (in
```

Comment:
Strict mode is already enabled in ES6 for module code.

Reply:
For ES6 modules, not for commonJS modules.

In a node module without strict mode doing
`foo = 'bar'`
Without declaring a variable first creates a `foo` property on the global object, i.e `global.foo`, same as like the window object in the browser.

Reply:
Absolutely, although now we are in Node 8.5.x territory it's entirely plausible to start using .mjs, so it would be worthwhile to say that the use of 'use strict' depends on context.